src/test/java/RegistrationTests.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RegistrationTests extends BaseTest{
+
+    public void successfulRegistrationMessage() {
+        String messageExpected = "We've sent a confirmation link to the email. Please continue by clicking on it";
+        WebElement messageActual = driver.findElement(By.xpath("//div[@class='login-wrapper']//div[@class='messages']"));
+        String message = messageActual.getText();
+        System.out.println(message);
+        Assert.assertEquals(message, messageExpected);
+    }
+    @Test
+    public void registrationNavigation() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.